lekko_client/evaluation/evaluation.py

```diff
@@ -0,0 +1,60 @@
+from dataclasses import dataclass
+from typing import List, Optional
+
+from google.protobuf.any_pb2 import Any
```

Comment:
nit: maybe it's just me, but when I see `Any` I think that it's `typing.Any`. Maybe import it as `ProtoAny`? Or import package and use it as `any_pb2.Any`?